crossbeam-deque/src/deque.rs

```diff
@@ -693,6 +693,47 @@ impl<T> Stealer<T> {
     /// assert_eq!(w2.pop(), Some(2));
     /// ```
     pub fn steal_batch(&self, dest: &Worker<T>) -> Steal<()> {
+        self.steal_batch_with_limit(dest, MAX_BATCH)
+    }
+
+    /// Steals no more than `limit` of tasks and pushes them into another worker.
+    ///
+    /// How many tasks exactly will be stolen is not specified. That said, this method will try to
+    /// steal around half of the tasks in the queue, but also not more than the given limit.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use crossbeam_deque::Worker;
+    ///
```

Comment:
`usize::MAX` (associated constant) is not available for older compilers, I would suggest adding the following hidden import to allow testing doctest on those versions.

```suggestion
    /// # use std::usize;
    ///
```

Reply:
changed to use `std::usize::MAX` directly to avoid an extra import.